test/org/zaproxy/zap/authentication/UsernamePasswordAuthenticationCredentialsTest.java

```diff
@@ -0,0 +1,115 @@
+package org.zaproxy.zap.authentication;
```

Comment:
- The file name should end with UnitTest (to be run by default in Travis builds).
 - It's missing the license header.